app/controllers/chargeback_controller.rb

```diff
@@ -725,6 +725,21 @@ def get_tags_all(category)
     classification.entries.each { |e| @edit[:cb_assign][:tags][e.id.to_s] = e.description } if classification
   end
 
+  def get_sub_tenants(tenant, parent)
+    data_tenant = []
+    tenant.all_subtenants.each do |subtenant|
```

Comment:
@aljesusg we need also include 'projects'. Projects are same as tenants but projects cannot have childrens.